<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>571</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="decadesCheckbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Decades</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="yearsCheckbox">
       <property name="text">
        <string>Years</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="quartersCheckbox">
       <property name="text">
        <string>Quarters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="seasonsCheckbox">
       <property name="text">
        <string>Seasons</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="monthsCheckbox">
       <property name="text">
        <string>Months</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="weeksCheckbox">
       <property name="text">
        <string>Weeks</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QGridLayout" name="chartBox">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="eventsLayout">
     <item>
      <widget class="QCheckBox" name="usHolidaysCheckBox">
       <property name="text">
        <string>US Holidays</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Company reports</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Dividends</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Duration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="durationComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentText">
          <string>Select Duration</string>
         </property>
         <item>
          <property name="text">
           <string>Select Duration</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>All</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20 years</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10 years</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5 years</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 year</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 quarter</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 month</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 week</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="updateLayout">
       <item>
        <widget class="QLabel" name="updateLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="loadingProgressBar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>6</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>6</height>
          </size>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QGridLayout" name="returnsChartBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
